### DISCUSSION TITLE
Is the title "Section 6: Attributes" correct?

### DISCUSSION THREAD

**Discussion by yuki-kimoto** [Ovid/Cor#67] [Q&A] [answered]
Is the title "Section 6: Attributes" correct?

https://github.com/Ovid/Cor/blob/master/rfc/attributes.md

The content seems like the description of "Fields".

The word "Attribute" seems to be used for ":param", ":reader".

https://github.com/Ovid/Cor/blob/master/rfc/attributes.md#623-field-attributes

**Comment by Ovid** [comment]
Good catch! I should probably fix that. The term "attribute" is a holdover from the Moo/se influence on the Corinna project. While discussing how to declare the instance variables for the clss, `attr` was discarded because the term "attribute" is so heavily overloaded and might be confusing or misleading. (Historical note: this is also why Perl's "roles" are called "roles" and not "traits" as they are in most languages which implements them).

**Reply by yuki-kimoto** [reply]
Thanks. I wait for the fix.

**Comment by Ovid** [comment, marked as answer]
This is now renamed to "Fields" because that's what it's describing. The filename has been left unchanged.

**Reply by yuki-kimoto** [reply]
Thanks!
